Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -4,11 +4,14 @@
 public class BuddyStats : MonoBehaviour
 {
 	public Stat statType = Stat.Invalid;
-	[SerializeField] int startingApples = 10;
-	[SerializeField] int apples = 0;
+	[SerializeField] bool canDecreaseStat = true;
```

Comment:
This is also poorly named since it's just a debug variable. It should be changed to something like `disableStatEffects`, and you should use a `[Header()]` attribute to split debug settings off from the main settings.